pkg/iac-providers/kustomize/v3/load-dir.go

```diff
@@ -0,0 +1,147 @@
+package kustomizev3
+
+import (
+	"errors"
+	"path/filepath"
+
+	k8sv1 "github.com/accurics/terrascan/pkg/iac-providers/kubernetes/v1"
+	"github.com/accurics/terrascan/pkg/iac-providers/output"
+	"github.com/accurics/terrascan/pkg/utils"
+	"go.uber.org/zap"
+	"sigs.k8s.io/kustomize/api/filesys"
+	"sigs.k8s.io/kustomize/api/krusty"
+)
+
+const (
+	kustomizedirectory string = "kustomize_directory"
+)
+
+// LoadIacDir loads the kustomize directory
+func (k *KustomizeV3) LoadIacDir(absRootDir string) (output.AllResourceConfigs, error) {
```

Comment:
It is a sincere request to add comments at function level and inside the function as well, so as to help the user understand what's going on. A new reader may not be able to immediately understand what task is being achieved in the function